status-im/status-legacy
# Pending stickers status maintains after app restart

fixes #9754

### Summary
If stickers purchase wasn't confirmed yet, user will see waiting spinner even after restart

**NOTE**: `status-go` [PR](https://github.com/status-im/status-go/pull/1830) should be merged first and `status-go-version.json` updated before merge of this one.


### Review notes
When user purchases a stickerpack, new`pending-stickers` record added to multiaccount to survive restart (Related changes in status-go: https://github.com/status-im/status-go/pull/1830).
After app start we run `set-pending-timeout-fx` again to make sure that pending stickers will be processed

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
Stickers purchasing



### Steps to test
- Open Status
- open available stickers
- sign transaction to purchase Tozemoon stickerpack and immediately switch off the internet
- restart the app 
- turn on the internet and quickly open available stickers page
- Tozemoon stickerpack should be pending and will change to available in a few seconds (current timeout for checking pending stickers is 10s)


status: ready
